src/components/visualizations/chart/highcharts/plugins/adjustTickAmount.ts

```diff
@@ -43,29 +44,34 @@ function isUserSetExtremesOnAnyAxis(chart: Highcharts.Chart): boolean {
  *      0: it aligns
  *      1: move zero index to right
  */
-function getDirection(primaryAxis: IHighchartsAxisExtend, secondaryAxis: IHighchartsAxisExtend): number {
-    const { tickPositions: primaryTickPosition } = primaryAxis;
-    const { tickPositions: secondaryTickPosition } = secondaryAxis;
-
-    const primaryZeroIndex = primaryTickPosition.indexOf(0);
-    const secondaryZeroIndex = secondaryTickPosition.indexOf(0);
-
-    // no need to align zero on axes without zero or aligned already
-    if (
-        isNil(primaryZeroIndex) ||
-        isNil(secondaryZeroIndex) ||
-        primaryZeroIndex < 0 ||
-        secondaryZeroIndex < 0 ||
-        primaryZeroIndex === secondaryZeroIndex
-    ) {
+export function getDirection(
+    primaryAxis: IHighchartsAxisExtend,
+    secondaryAxis: IHighchartsAxisExtend,
+): number {
+    if (isNil(primaryAxis) || isNil(secondaryAxis)) {
+        return ALIGNED;
+    }
+
+    const { tickPositions: primaryTickPositions = [] } = primaryAxis;
+    const { tickPositions: secondaryTickPositions = [] } = secondaryAxis;
```

Comment:
@ptquang86 , I have to add default value back in here.
At some first stages, Highcharts does not initialize tick positions for Y axis, so it could be `undefined` and we cannot call `indexOf`.